### DISCUSSION TITLE
Event Stream: allow to POST + body?

### DISCUSSION THREAD

**Discussion by wivaku** [Kong/insomnia#6464] [Ideas]
For GraphQL subscription that uses SSE (Event Stream) I need to provide the query in the body.
Currently Insomnia only allows entering the URL for an SSE request, not a body.

Are there plans to add this to Insomnia?

**Comment by Liu-Da** [comment]
I also face the same problem when using insomnia to call OpenAI (GPT3.5) /v1/chat/completions API and returns SSE Response. This is my biggest obstacle currently in switching from postman to insomnia.
